app/backend/src/RPG/routes/router.ts

```diff
@@ -0,0 +1,19 @@
+import { Router } from "express";
```

Comment:
if we have a component with routes folder, we can organize all routes that comes from RPG, if this expands but cover different services we can create `> src > components > rolePlayingGame > routes > api.ts ` for example, and centralize that into a `index.ts` so that express can import all at once. Depending of the component, the routes can get really messy.

Reply:
this is already done in the folder `> src > routes`